Investing - Vanguard poor use of Yubikey (FIDO) noted

PeterTheobald opened the pull request:
Updated Vanguard entry to note that use of FIDO device like Yubikey is trivially bypassed by simply checking a box for "I forgot my device please use SMS instead". SMS cannot be disabled, thus the security of Yubikey is not actually enforced. Tested as of Jan 2018.

PeterTheobald committed:
Investing - Vanguard poor use of Yubikey (FIDO) noted

Updated Vanguard entry to note that use of FIDO device like Yubikey is trivially bypassed by simply checking a box for "I forgot my device please use SMS instead". SMS cannot be disabled, thus the security of Yubikey is not actually enforced. Tested as of Jan 2018.

PeterTheobald committed:
Updated investing - added exception describing Vanguard limitation

Vanguard doesn't allow U2F/FIDO (Yubikey) without SMS option

jamcat22 committed:
Update investing.yml

Old statement could be interpreted as SMS 2FA being always on. Also clarified that phone call 2FA can be used instead of SMS.

PeterTheobald committed:
updating Bank of America

There is no hardware 2FA. Not sure why it was set to Yes, but I checked BankOfAmerica.com thoroughly. Only SMS.

Carlgo11 reviewed (changes requested):
Try and keep changes to one per PR.
Remove the BOA change from this PR and move it to another PR.
I suggest you use separate branches instead of pushing to your master branch. 😉

PeterTheobald committed:
Reverted Bank of America change

moving it to its own branch

PeterTheobald committed:
Updated Bank of America

Checked Bank of America website thoroughly - no mention of any hardware option. Fixed entry to hardware: No

PeterTheobald committed:
Merge pull request #1 from PeterTheobald/PeterTheobald-bankofamerica

Updated Bank of America
